HelpMyStreet.Utils/HelpMyStreet.Utils/Enums/CommunicationJobTypes.cs

```diff
@@ -14,6 +14,7 @@ public enum CommunicationJobTypes
         PostYotiCommunication = 6,
         SendTaskReminder = 7,
         InterUserMessage = 8,
-        NewCredentials = 9
+        NewCredentials = 9,
+        TaskDetail = 10
```

Comment:
Sorry, I've already used 10 and 11 in this PR: https://github.com/HelpMyStreet/nuget-package/pull/124/files (has been pending Jav's approval for a while!)  Could TaskDetail be 12?